app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        cp, old_file, new_file = command.split()
+        if cp != "cp" or old_file == new_file:
+            raise ValueError
+    except ValueError:
+        raise ValueError("Write a command correctly. "
```

Comment:
why do u need to use ValueError twice? u can catch a several exceptions in ur except block

Reply:
It seems to me if I delete raise in if condition, then I won't get raise error when ```cp``` var won't be equal to "cp", thus my command will be able to execute for example with the "move" command

> why do u need to use ValueError twice? u can catch a several exceptions in ur except block